app/change_sets/workflow_note_change_set.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class WorkflowNoteChangeSet < Valkyrie::ChangeSet
+  validates :author, :note, presence: true
```

Comment:
The property definition below says note isn't required (which is what I'd expect), but this is validating its presence.  Why define the property as non-required and then enforce its presence?